### PR TITLE
14348 cluster search

### PR DESCRIPTION
**Pull request by unodgs**
CR

### CODE REVIEW

File: src/views-components/data-explorer/renderers.tsx [hunk 1]
```diff
@@ -224,6 +224,28 @@ const renderNodeInfo = (data: string) => {
...
+const clusterColors = [
+    ['#f44336', '#fff'],
+    ['#2196f3', '#fff'],
+    ['#009688', '#fff'],
+    ['#cddc39', '#fff'],
+    ['#ff9800', '#fff']
+];
+
+export const ResourceCluster = (props: { uuid: string }) => {
+    const p = props.uuid.indexOf('-');
```
Comment (burzomir) [inline src/views-components/data-explorer/renderers.tsx]: Give meaningful names for variables. Extract magic numbers like 5 to constants.

File: src/components/data-explorer/data-explorer.tsx [hunk 2]
```diff
@@ -110,14 +112,18 @@ export const DataExplorer = withStyles(styles)(
...
                             rowsPerPage={rowsPerPage}
                             rowsPerPageOptions={rowsPerPageOptions}
                             page={this.props.page}
                             onChangePage={this.changePage}
                             onChangeRowsPerPage={this.changeRowsPerPage}
-                            component="div" />
+                            component="div" /> : <Button
+                                variant="text"
+                                size="medium"
+                                onClick={() => this.loadMore(this.props.page)}
```
Comment (burzomir) [inline src/components/data-explorer/data-explorer.tsx]: onClick={this.loadMore}

File: src/components/data-explorer/data-explorer.tsx [hunk 3]
```diff
@@ -131,6 +137,10 @@ export const DataExplorer = withStyles(styles)(
             this.props.onChangeRowsPerPage(parseInt(event.target.value, 10));
         }
 
+        loadMore = (page: number) => {
+            this.props.onLoadMore(page + 1);
```
Comment (burzomir) [inline src/components/data-explorer/data-explorer.tsx]: loadMore = () => this.props.onLoadMote(this.props.page + 1)

File: src/store/search-results-panel/search-results-middleware-service.ts [hunk 4]
```diff
@@ -15,32 +15,66 @@ import { OrderDirection, OrderBuilder } from '~/services/api/order-builder';
...
+
+            api.dispatch(updateResources(list.items));
+            api.dispatch(criteriaChanged
+                ? setItems(list)
+                : appendItems(list)
+            );
+
         } catch {
-            api.dispatch(couldNotFetchWorkflows());
+            api.dispatch(couldNotFetchSearchResults());
```
Comment (kowalczykpawel7) [inline src/store/search-results-panel/search-results-middleware-service.ts]: better name :D

**Pull request merged**